packages/hash/api/src/graph/system-types.ts

```diff
@@ -68,59 +62,54 @@ export let SYSTEM_TYPES: {
      */
     dummy: EntityTypeModel;
   };
-  /**
-   * @todo: bring back link types as entity types
-   * @see https://app.asana.com/0/1202805690238892/1203361844133477/f
-   */
-  // linkType: {
-  //   // HASHInstance-related
-  //   admin: LinkTypeModel;
+  entityLinkType: {
```

Comment:
I think that `linkEntityType` is a more appropriate name

Reply:
Addressed in https://github.com/hashintel/hash/pull/1400/commits/bb71e823638883092a9d43866dba97b7bb8918b6